### PR TITLE
Enable different fonts and colors configuration on multiline labels

### LINKED ISSUE #92: How to modify data labels?
I have played with the treemap and got to this result:
image

But what I aim at is this:
image

Is is possible to modify the labels and set different sizes (maybe via a plugin?)
Also, labels currently do not wrap (unless we wrap them manually by using arrays). You think this is something we have to stick with or it would be possible to also automatically wrap them somehow?

### PR DESCRIPTION
**Pull request by stockiNail**
Fix #92

treemapmultilinefonts

### TODO

- [x] test cases
- [x] types definitions changes
- [x] documentation
- [x] samples

### CODE REVIEW

**Comment by stockiNail** [comment]
@kurkle @LeeLenaleee if you agree with this implementation, what do you think to have the same in annotation plugin? This could fix an open issue where we suggested to use a label by a canvas.

File: docs/.vuepress/config.js [hunk 1]
```diff
@@ -48,6 +48,7 @@ module.exports = {
         '/samples/': [
           'basic',
           'labels',
+          'labelsFontsAndColors',
```
Comment (kurkle) [inline docs/.vuepress/config.js:51]: maybe not just yet, but when 3rd sample is created for labels, those should be placed in a folder

**Review by kurkle — APPROVED** [review]
off topic: should improve the test-utils spriting with sizes and colors :)

File: docs/.vuepress/config.js (hunk 1 above)
Comment (kurkle) [reply docs/.vuepress/config.js:51]: yes, always :)

**Comment by stockiNail** [comment]
@kurkle can I merge or do you want to wait?

**Pull request merged**